src/lib.rs

```diff
@@ -362,9 +362,31 @@ fn builder_for_struct(ast: syn::MacroInput) -> quote::Tokens {
         )
     };
 
+    let builder_struct_doc = format!(
+        "Builder for {s}.\n\
+        \n\
+        # Examples\n\
+        \n\
+        ```\n\
```

Comment:
I can find the documentation in `cargo expand --example missing_docs` ([examples/missing_docs.rs](https://github.com/colin-kiegel/rust-derive-builder/blob/ca762dbfac5733acb6e730e52675d9222f8afc39/examples/missing_docs.rs)). I expected this to behave like a foreign crate, but I can double check. :-)